doc/Command-Reference.md

```diff
@@ -7961,8 +7962,292 @@ Go Back To [Beginning of the document](#) or [Beginning of this section](#waterm
+    cpu-report azure/sonic-cpu-report --default-reference 1.0.0
+  ```
+
+**sonic-package-manager repository remove**
+
+This command will remove an entry from the package database. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager repository remove [OPTIONS] NAME
+
+    Remove package from database.
+
+  Options:
+    --help  Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager repository remove cpu-report
+  ```
+
+**sonic-package-manager install**
+
+This command pulls and installs package on SONiC host. *NOTE*: this command requires elevated (root) privileges to run.
+
+- Usage:
+  ```
+  Usage: sonic-package-manager install [OPTIONS] [PACKAGE_EXPR]
+
+    Install package
+
+  Options:
+    --enable                        Set the default state of the feature to
+                                    enabled and enable feature right after
+                                    installation. NOTE: user needs to execute
+                                    "config save -y" to make this setting
+                                    persistent
+
+    --default-owner [local|kube]    Default owner configuration setting for a
+                                    feature
+
+    --from-repository TEXT          Fetch package directly from image registry
+                                    repository NOTE: This argument is mutually
+                                    exclusive with arguments: [from_tarball,
+                                    package_expr].
+
+    --from-tarball FILE             Fetch package from saved image tarball
+                                    NOTE: This argument is mutually exclusive
+                                    with arguments: [from_repository,
+                                    package_expr].
+
+    -f, --force                     Force operation by ignoring failures
+    -y, --yes                       Automatically answer yes on prompts
+    -v, --verbosity LVL             Either CRITICAL, ERROR, WARNING, INFO or
+                                    DEBUG
+
+    --skip-cli-plugin-installation  Do not install CLI plugins provided by the
+                                    package on the host OS. NOTE: In case when
+                                    package /cli/mandatory field is set to True
+                                    this option will fail the installation.
+
+    --help                          Show this message and exit.
+  ```
+- Example:
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install dhcp-relay==1.0.2
+  ```
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install --from-repository azure/sonic-cpu-report:latest
+  ```
+  ```
+  admin@sonic:~$ sudo sonic-package-manager install --from-tarball sonic-docker-image.gz
+  ```
+
+**sonic-package-manager uninstall**
+
+This command uninstalls package from SONiC host. *NOTE*: this command requires elevated (root) privileges to run.
```

Comment:
Does this bring back, original copy for images that are part of install ?

Reply:
No, that would mean the switch has to save previous docker images and may take more disk space.
User can uninstall and install again older version if he would like to rollback.

This was not previously discussed as a requirement, so may I suggest not to add more functionality as part of this PR?
Besides, user is not blocked to uninstall and install or downgrade to previous versions of packages.

This PR is phase 1 and trying to provide minimum to get the 3rd party docker up and runnning on SONiC, while the rollback (previous comment) and original images retention seems like somthing we can build on top of the minimum which we defined for phase 1.